frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js

```diff
@@ -14,41 +15,48 @@ const Review = ({
   valid,
   handleSubmit,
   onFormReview,
-}) => (
-  <>
-    {reviewed
-      && (
-      <Alert noIcon className="margin-y-4" type="success">
-        <b>Success</b>
-        <br />
-        Your review of this report was successfully submitted
-      </Alert>
-      )}
-    <h2>Review and approve report</h2>
-    <div className="smart-hub--creator-notes" aria-label="additionalNotes">
-      <p>
-        <span className="text-bold">Creator notes</span>
-        <br />
-        <br />
-        { additionalNotes || 'No creator notes'}
-      </p>
-    </div>
-    <Form className="smart-hub--form-large" onSubmit={handleSubmit(onFormReview)}>
-      <Fieldset className="smart-hub--report-legend smart-hub--form-section" legend="Review and submit report">
-        <Label htmlFor="managerNotes">Manager notes</Label>
-        <Textarea inputRef={register} id="managerNotes" name="managerNotes" />
-      </Fieldset>
-      <Label htmlFor="status">Choose report status</Label>
-      <Dropdown id="status" name="status" defaultValue="" inputRef={register({ required: true })}>
-        <option name="default" value="" disabled hidden>- Select -</option>
-        {managerReportStatuses.map((status) => (
-          <option key={status} value={status}>{_.startCase(status)}</option>
-        ))}
-      </Dropdown>
-      <Button type="submit" disabled={!valid}>Submit</Button>
-    </Form>
-  </>
-);
+}) => {
+  // FIXME: Use 'register', 'handleSubmit' from useFormContext rather than passed props
```

Comment:
Since `useFormContext` has now been added in other places, can we remove register and handleSubit from the props now?

Reply:
I had created a ticket, #289, for switching to `useFormContext` because I thought that was out of scope of this ticket, and could be done in another PR. It looks like that #289 was closed, but I still think refactoring to use `useFormContext` belongs in a separate pull request. Does that make sense?

Reply:
It was closed because it was completed, so maybe it's just a matter of merging in adhoc/main to this branch.

Reply:
Looking at main, I can see these [props still exist](https://github.com/adhocteam/Head-Start-TTADP/blob/main/frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js#L13), and I was pretty sure that last few times I merged and dealt with the conflicts.

Anyway, I can take another look at switching these to `useFormContext`. I can't recall if there were other reasons I didn't just do the whole enchilada here, but I'll give it a shot.

Reply:
Ah, yeah, looks like it's just the final page that needs to be updated still. For instance, this has been updated:

https://github.com/adhocteam/Head-Start-TTADP/blob/main/frontend/src/pages/ActivityReport/Pages/activitySummary.js#L24-L30

I'm fine with reopening #289 (or opening a new issue) to move these pages over if this'll take too long

Reply:
Oh, I think one reason was that I also noticed that `onFormSubmit` was being passed down multiple levels, and could use a Context Provider/Consumer of its own. Would you want me to do that in this PR as well, create a ticket, or not worry about it?

Reply:
That feels like its own ticket to me

Reply:
Okay, will create ticket.

In the meantime, Review-related components have been updated to take advantage of `useFormContext`.